spec.html

```diff
@@ -1016,6 +1016,117 @@ <h1>Example Cyclic Module Record Graphs</h1>
         </tbody>
       </table>
     </emu-table>
+
+    <p><ins>
+      Alternatively, Let's consider a failure case, where _C_ fails execution andd returns an error.  When that happens, AsyncModuleExecutionRejected is called and _C_.[[AsyncEvaluating]] is set to *false*. We then set the evaluation error of the module to the error returned by executing _C_. We then propogate this error to all of the AsyncParentModules by performing AsyncModuleExeutionRejected on each of them. The fields of the updated modules are as given in <emu-xref href="#table-module-graph-cycle-async-fields-7"></emu-xref>.
```

Comment:
It could also be worth clarifying that `_C_ fails execution before _B_ has finished executing and returns an error`. Since `_B_` could have finished first on this graph.